### ISSUE TITLE
Agenda for 2nd AllERCDevs (EU/US Friendly) 2023-06-01 UTC1500

### ISSUE THREAD

**Issue by xinbenlv** [ercref/AllERCDevs#2]
A placeholder for signing up the agenda for second session of AllERCDevs

The time of event will be 

- 2023-06-01 UTC 1500-1559 Thursday*, which translates to
 
- 08:00-08:59 on 2023-06-01 Thursday in US Pacific Time.
- 11:00-11:59 on 2023-06-01 Thursday in US Eastern Time.
- 23:00-23:59 on 2023-06-01 Thursday in Singapore.

The event will be hosted on Zoom, a link will be posted on "all-erc-devs" channel on this [Discord Server](https://discord.gg/2arW34S6J7)

Agenda will be updated and please feel free to comment to signup your agenda items.

### Main Session (1h)

1. [ ] (15-20min) *Featured Talk*:  [Wallet Test Framework(*WTF*)](https://wtf.allwallet.dev/) by @SamWilsn 
2. (10min) *Demo Show & Tell* (5-10min per slot)
- [ ] (5m)[HexLink](https://hexlink.io/) by @pette1999
- [ ] (5m)[TurboETH](https://www.turboeth.xyz/) by @kamescg 
4. (30min) *ERC Peer Review Requests* (5-10min per slot)
- TBD: (5-10min) [EIP6982 Efficient Default Lockable NFTs](https://github.com/ethereum/EIPs/blob/82372cbae88746c496ce1fbd4524862606de7421/EIPS/eip-6982.md) and [EIP6454](https://eips.ethereum.org/EIPS/eip-6454) by @sullof
- (5-10min) https://github.com/ethereum/EIPs/pull/7093 by @odysseus0 
5. Ad Hoc Sharing / Pre-ERC sharing
- (5-10min) Unruggable Protocol by @nxt3d

### Social Hour (30m)

- TODO decide social hour topics

Please feel free to signup the agenda slot by commenting Below.

## Meeting Link
Zoom Meeting Link: https://us06web.zoom.us/j/88694529887?pwd=NDRWYXlhejBvcXFVSzEzRFluT0Jmdz09

Meeting ID: 886 9452 9887
Passcode: 338406

## Notes and transcription
AI Generated Notes by Airgram.io: https://app.airgram.io/7065496599555735553/share/b8f31df0008c11ee91af0a2cbd030b4d?from=A91fc19caf6ac11eda1a80a2cbd030b4d-7065496599555735553
AI Generated Notes by Fireflies.ai: https://app.fireflies.ai/view/AllERCDevs-EU-US-time-friendly-session-::Ep4E220xTz

Full Transcription
[AllERCDevs (EU_US time friendly session).docx](https://github.com/ercref/AllERCDevs/files/11630492/AllERCDevs.EU_US.time.friendly.session.docx)

**Comment by samwilson** [comment]
I think you might mean a different @samwilson :-)

**Comment by xinbenlv** [comment]
@samwilson yeah, it's actually @SamWilsn

**Comment by xinbenlv** [comment]
Per #1 
> I may talk about [EIP6982 Efficient Default Lockable NFTs](https://github.com/ethereum/EIPs/blob/82372cbae88746c496ce1fbd4524862606de7421/EIPS/eip-6982.md) and [EIP6454](https://eips.ethereum.org/EIPS/eip-6454) which move in the same space, but with different targets in mind

Hey @sullof, it was a pity you didn't make the first one, do you want to signup the second one?

**Comment by sullof** [comment]
@xinbenlv On the Discord server there was so little activity that I thought that the event would never happen and I got distracted. I am sorry about that. Next time, I suggest you use a tool for events instead of a GitHub issue that is very hard to follow if you follow dozens of repos like me.
BTW, do you have a recording of the event?

**Comment by xinbenlv** [comment]
The recording of first session is published in https://youtu.be/GNVSOgZ7g_Y
